src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+
+    <body>
+      <div class="container">
+        <article class="card" data-qa="card">
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__rate">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star stars__star--inactive"></div>
+            </div>
```

Comment:
Check the requirements - `Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational`.
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```